libbeat/_meta/fields.common.yml

```diff
@@ -15,6 +15,11 @@
       description: >
         The hostname as returned by the operating system on which the Beat is
         running.
+      metricbeat.format: url
```

Comment:
Ok, lets keep it then and figure out a way in the long run how we can make this nicer. Perhaps we can somehow have it also in metricbeat in the future and merge the fields values somehow. 

Do you think this will also happen to other fields then only `beat.hostname` or is this hostname specific?